attestation-service/rvps/src/store/local_json/mod.rs

```diff
@@ -0,0 +1,52 @@
+use std::{collections::HashMap, fs};
+
+use super::Store;
+use crate::ReferenceValue;
+use anyhow::Result;
+use log::warn;
+use serde::Deserialize;
+use serde_json::Value;
+
+const FILE_PATH: &str = "/opt/confidential-containers/attestation-service/reference_values.json";
+
+pub struct LocalJson {
+    file_path: String,
+}
+
+#[derive(Deserialize, Default)]
+struct Config {
+    file_path: Option<String>,
+}
+
+impl LocalJson {
+    pub fn new(config: Option<Value>) -> Result<Self> {
+        let config = match config {
+            Some(v) => serde_json::from_value(v)?,
+            None => Config::default(),
+        };
+        Ok(Self {
+            file_path: config.file_path.unwrap_or_else(|| {
+                warn!("json file storage backend use default path {FILE_PATH}");
+                FILE_PATH.into()
+            }),
+        })
+    }
+}
+
+impl Store for LocalJson {
+    fn set(&mut self, name: String, rv: ReferenceValue) -> Result<Option<ReferenceValue>> {
+        let file = fs::read(&self.file_path)?;
+        let mut dic: HashMap<String, ReferenceValue> = serde_json::from_slice(&file)?;
+        let res = dic.insert(name, rv);
+        let contents = serde_json::to_vec(&dic)?;
```

Comment:
shouldn't the file be `ReferenceValue[]`, instead of a not a top level key/value  object?

Reply:
Ah. Yes. let me fix this.